File: argv/c/argv.c

#include <stdio.h>

int main(int argc, char *argv[])
{
	int i;

	printf("%d arguments\n", argc);

	for (i = 0; i < argc; i++) {
		printf("%d: '%s'\n", i, argv[i]);
	}

	return 0;
}

File: argv/go/argv.go
package main

import "os"
import "fmt"

func main() {
	len_Args := len(os.Args)
	fmt.Printf("%v arguments", len_Args)

	for i := 0; i < len_Args; i++ {
		fmt.Printf("%v: '%v'\n", i, os.Args[i])
	}
}
